guides/common/modules/proc_installing-capsule-server-packages.adoc

```diff
@@ -28,4 +34,28 @@ ifndef::satellite[]
 ----
 # {package-install} foreman-proxy-content
 ----
+=======
```

Comment:
I think this line is breaking rendering.

Reply:
Looks like it, I've pushed the fix.